lib/elixirconf_chat/chat.ex

```diff
@@ -35,7 +35,9 @@ defmodule ElixirconfChat.Chat do
   Fetches a Room by ID.
   """
   def get_room(room_id) do
-    room = Repo.get(Room, room_id)
+    room =
+      Repo.get(Room, room_id)
+      |> Repo.preload(:messages)
```

Comment:
This might be personal preference but I think its a bit cleaner to pipe the first argument if you're using pipes anyway.

```suggestion
      Room
      |> Repo.get(room_id)
      |> Repo.preload(:messages)
```